backend/app/usecases/bot.py

```diff
@@ -131,6 +133,16 @@ def create_new_bot(user_id: str, bot_input: BotInput) -> BotOutput:
                 chunk_size=chunk_size,
                 chunk_overlap=chunk_overlap,
             ),
+            generation_params=GenerationParamsModel(
```

Comment:
Please care the null input case using default value (See the comment on route/schemas)